db/fixtures/conditions.yml

```diff
@@ -0,0 +1,20 @@
+---
+- :name: 3366fc29-d457-4c64-a648-e2a5686600cd
+  :description: a condition
+  :modifier: allow
+  :expression:  |
+     --- !ruby/object:MiqExpression
+       exp:
+         IS NOT NULL:
+           field: Vm-name
+           value: ""
+       context_type:
+  :towhat: Vm
+  :file_mtime:
+  :created_on:
+  :updated_on:
+  :guid: 3366fc29-d457-4c64-a648-e2a5686600cd
+  :filename:
+  :applies_to_exp:
+  :miq_policy_guid: c5ceab70-da38-11e5-9cae-0242782a2608
```

Comment:
I'm not sure why a condition has a miq_policy_id but i'm pretty sure I instead I need to seed conditions_miq_policies one to many association instead.